### PR TITLE
Mobile webpage1

### PR DESCRIPTION
**Pull request by Lockless404**
Completed phase 2 of the project. Added the 'My recent works' and 'about me pages'. Had a problem at the end where the list-syle-images would only work my inner lists even though I moved it around to all the list, help would be greatly appreciated.

### CODE REVIEW

File: portfolio.css [hunk 1]
```diff
@@ -135,3 +139,349 @@ h2 {
...
+  font-size: 15px;
+  font-style: normal;
+  font-weight: 500;
+  line-height: 20px;
+  text-align: left;
+  background-color: #ffffff3d;
+}
+
+.cardBtn {
+  padding: 15px;
```
Comment (afanimartin) [inline portfolio.css:365]: Please follow the button styling: Active, Hover, and Pressed (see design). It's always a good practice to follow a given design.

File: portfolio.html [hunk 2]
```diff
@@ -27,5 +27,113 @@ <h2>I’am a Software Developer</h2>
...
+    <p class="secondPara">A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a standard dummy text.</p>
+    <ul class="languageList">
+      <li class="languageList css">css</li>
+      <li class="languageList html">html</li>
+      <li class="languageList bootstrap">bootstrap</li>
+      <li class="languageList ruby">Ruby</li>
+    </ul>
+    <button class="seeButton">See Project</button>
+  </section>
+  <section class="projects">
```
Comment (afanimartin) [inline portfolio.html:43]: Please add some breathing room to the section by defining top, left, right, and bottom margins.

File: portfolio.html [hunk 3]
```diff
@@ -27,5 +27,113 @@ <h2>I’am a Software Developer</h2>
...
+    <div class="cards">
+      <h3>Profesional Art Printing Data</h3>
```
Comment (afanimartin) [inline portfolio.html:45]: The `See Project` button is wider for this card than the previous card. It's a good idea to follow the given design and also make sure there's uniformity in the flow of items on the page.
Screenshot from 2021-09-17 16-42-19

**Review by afanimartin — CHANGES_REQUESTED** [review]
Hi @Lockless404, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

### Martin Afani
#### Slack: whiz25

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by rukundoeric — APPROVED** [review]
Hi @Lockless404,

_Excellent job on this project. Keep it up 🎆_

_Congratulations! 🎉  on the splendid job 👍
You can now merge this PR and submit the project completion form 🚀 . I wish you all the very best in your next tasks 🥇_


Cheers and Happy coding!👏👏👏

__Feel free to leave any questions or comments in the PR thread if something is not 100% clear.__

------
>As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Pull request merged**